examples/media/media.html

```diff
@@ -0,0 +1,211 @@
+          if (newState) {
+            this.onChange(newState);
+            return true;
+          }
+          return false;
+        }
+
+        _addMedia(type) {
+          const src = window.prompt('Enter a URL');
+          if (!src) {
+            return;
+          }
+
+          const entityKey = Entity.create(type, 'IMMUTABLE', {src});
+
+          return MediaUtils.insertMedia(this.state.editorState, entityKey, ' ');
+        }
+
+        _addAudio() {
+          this.onChange(this._addMedia('audio'));
+        }
+
+        _addImage() {
+          this.onChange(this._addMedia('image'));
+        }
+
+        _addVideo() {
+          this.onChange(this._addMedia('video'));
+        }
+
+        render() {
+          return (
+            <div style={styles.root}>
+              <div style={{marginBottom: 10}}>
+                Use the buttons to add audio, image, or video.
+              </div>
+              <div style={{marginBottom: 10}}>
+                Here are some local examples that can be entered as a URL:
+                <ul>
+                  <li>media.mp3</li>
+                  <li>media.png</li>
+                  <li>media.mp4</li>
+                </ul>
+              </div>
+              <div style={styles.buttons}>
+                <button onMouseDown={this.addAudio} style={{marginRight: 10}}>
+                  Add Audio
+                </button>
+                <button onMouseDown={this.addImage} style={{marginRight: 10}}>
+                  Add Image
+                </button>
+                <button onMouseDown={this.addVideo} style={{marginRight: 10}}>
+                  Add Video
+                </button>
+              </div>
+              <div style={styles.editor} onClick={this.focus}>
+                <Editor
+                  blockRendererFn={mediaBlockRenderer}
+                  editorState={this.state.editorState}
+                  handleKeyCommand={this.handleKeyCommand}
+                  onChange={this.onChange}
+                  placeholder="Enter some text..."
+                  ref="editor"
+                />
+              </div>
+              <input
+                onClick={this.logState}
+                style={styles.button}
+                type="button"
+                value="Log State"
+              />
+            </div>
+          );
+        }
+      }
+
+      function mediaBlockRenderer(block) {
+        if (block.getType() === 'media') {
+          return {
+            component: Media,
+            props: {},
+          };
+        }
+
+        return null;
+      }
+
+      const Audio = (props) => {
+        return <audio controls src={props.src} style={styles.media} />;
+      };
+
+      const Image = (props) => {
+        return <img src={props.src} style={styles.media} />;
+      };
+
+      const Video = (props) => {
+        return <video controls src={props.src} style={styles.media} />;
+      };
+
+      const Media = (props) => {
```

Comment:
Thinking it might be good to wrap the output of this function in a `div`. I'm not sure how I feel about an example that potentially returns `display: inline` elements for block renderers.